taosctl/cmd_ui.py

```
@@ -0,0 +1,41 @@
+@click.option(
+    "--app-origin",
+    "app_origin",
+    help="Optional attribution shown to the user; defaults to the calling agent's name.",
+)
+@click.option("--json", "json_out", is_flag=True, help="Emit only the raw JSON response.")
+def notify_cmd(title: str, body: str, priority: str, app_origin: str | None, json_out: bool) -> None:
+    payload = {"title": title, "body": body, "priority": priority}
+    if app_origin:
+        payload["app_origin"] = app_origin
+    result = http_client.post("/api/ui/notify", json=payload)
+    click.echo(json.dumps(result, indent=2) if json_out else json.dumps(result, indent=2))
```

Ruff (0.15.12)

[warning] Useless `if`-`else` condition

(RUF034)